tests/unit_tests/test_helpers.py

```diff
@@ -0,0 +1,111 @@
+import unittest
+import logging
+import boto3
+import unittest
+from moto import mock_dynamodb2
+import logging.config
+from app.helpers import add_item, check_and_get_url_short, create_url, fetch_url
+logger = logging.getLogger(__name__)
+
+
+class TestDynamoDb(unittest.TestCase):
+    def setup(self):
+        logger.debug("Setting up Dynamo Db tests")
+        self.keys_and_urls = {}
+        self.testing_urls = []
+        self.table = self.create_fake_table()
+
+    @mock_dynamodb2
+    def create_fake_table(self):
+        logger.debug("Creating fake Dynamo Db Table")
+        region = 'eu-central-1'
+        dynamodb = boto3.resource('dynamodb', region)
+
+        def populate_table():
+            logger.debug("Populating the table using the create_url method")
+            for url in self.testing_urls:
+                uuid_key = create_url(self.table, url)
+                self.keys_and_urls[uuid_key] = url
+
+        table = dynamodb.create_table(
+            TableName='shorturl',
+            AttributeDefinitions=[
+                {
+                    'AttributeName': 'url',
+                    'AttributeType': 'S'
+                },
+                {
+                    'AttributeName': 'url_short',
+                    'AttributeType': 'S'
+                },
+                {
+                    'AttributeName': 'timestamp',
+                    'AttributeType': 'S'
+                },
+                {
+                    'AttributeName': 'epoch',
+                    'AttributeType': 'S'
+                },
+
+            ],
+
+            KeySchema=[
+                {
+                    'AttributeName': 'url_short',
+                    'KeyType': 'HASH'
+                },
+                {
+                    'AttributeName': 'url',
+                    'KeyType': 'HASH'
```

Comment:
Here you could update the yapf config as in the guideline, I fixed some config to improve the readability of complexe structure like these ones, it would make the code a bit more concise.